toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -140,10 +157,12 @@ func resolveGraphNodes(dependencyGraph *pkggraph.PkgGraph, inputSummaryFile, out
 		timestamp.StartEvent("clone graph", nil)
 
 		for _, n := range unresolvedNodes {
-			resolveErr := resolveSingleNode(cloner, n, toolchainPackages, fetchedPackages, prebuiltPackages, *outDir)
+			resolveErr := resolveSingleNode(cloner, n, downloadDependencies, toolchainPackages, fetchedPackages, prebuiltPackages, *outDir)
 			// Failing to clone a dependency should not halt a build.
 			// The build should continue and attempt best effort to build as many packages as possible.
 			if resolveErr != nil {
+				resolveErr = fmt.Errorf("failed to resolve graph node '%s':\n%w", n, resolveErr)
+				logger.Log.Warn(resolveErr)
```

Comment:
Since this is a warning, let's use the non-error capitalization:
```suggestion
				logger.Log.Warnf("Failed to resolve graph node '%s':\n%w", n, resolveErr)
```